app/src/main/res/values-iw

```diff
@@ -1 +0,0 @@
-values-he/
```

Comment:
Why was that deleted?

Reply:
Still not as it was before ;)

Reply:
Ah sorry, I've got some difficulties to put back this one. It should be ok now